bin/www

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env node
-
+PORT = 4000;
```

Comment:
This line can be placed in a ".env" file in the root of the project.
Then, because the npm package "dotenv" is installed and initialized, the environment variable can be read from process.env.